### PR TITLE
Enhancement/skale 3271 upgrade compatibility

### PR DESCRIPTION
**Pull request by yavrsky**

### CODE REVIEW

**Review by payvint — DISMISSED** [review]
Nice - approved!

File: scripts/test_upgrade.sh [hunk 1]
```diff
@@ -1,22 +1,26 @@
 #!/usr/bin/env bash
 
-VERSION=$(cat VERSION)
-DEPLOYED_VERSION=$(cat $TRAVIS_BUILD_DIR/DEPLOYED)
-DEPLOYED_DIR=$TRAVIS_BUILD_DIR/deployed-skale-manager/
+DEPLOYED_DIR=$GITHUB_WORKSPACE/deployed-skale-allocator/
+DEPLOYED_MANAGER_DIR=$GITHUB_WORKSPACE/deployed-skale-manager/
 
-git clone --branch $DEPLOYED_VERSION https://github.com/$TRAVIS_REPO_SLUG.git $DEPLOYED_DIR
+rm -rf $DEPLOYED_DIR && rm -rf $DEPLOYED_MANAGER_DIR
```
Comment (DimaStebaev) [inline scripts/test_upgrade.sh]: Is this line needed?

File: scripts/test_upgrade.sh [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
-cd $DEPLOYED_DIR
+cd $DEPLOYED_MANAGER_DIR
 yarn install
-npx oz push --network test --force || exit $?
 NODE_OPTIONS="--max-old-space-size=4096" PRODUCTION=true npx truffle migrate --network test || exit $?
-rm $TRAVIS_BUILD_DIR/.openzeppelin/dev-*.json
-cp .openzeppelin/dev-*.json $TRAVIS_BUILD_DIR/.openzeppelin
-cd $TRAVIS_BUILD_DIR
+cp data/test.json $DEPLOYED_DIR/scripts/manager.json
+cp data/test.json $GITHUB_WORKSPACE/scripts/manager.json
```
Comment (DimaStebaev) [inline scripts/test_upgrade.sh]: Why do we need this line?

File: scripts/test_upgrade.sh [hunk 3]
```diff
@@ -1,22 +1,26 @@
...
+cd $DEPLOYED_DIR
+yarn install
+NODE_OPTIONS="--max-old-space-size=4096" npx truffle migrate --network test || exit $?
+rm $GITHUB_WORKSPACE/.openzeppelin/dev-*.json
+cp .openzeppelin/dev-*.json $GITHUB_WORKSPACE/.openzeppelin
+cd $GITHUB_WORKSPACE
 
-kill $GANACHE_PID
\ No newline at end of file
+npx oz upgrade --network test --all || exit $?
```
Comment (DimaStebaev) [inline scripts/test_upgrade.sh:24]: You forgot to kill ganache with `kill $GANACHE_PID`

**Review by DimaStebaev — APPROVED** [review]

**Pull request merged**